deployment/terraform/firewall.tf

```diff
@@ -0,0 +1,47 @@
+#
+# API ALB security group resources
+#
+resource "aws_security_group_rule" "alb_http_ingress" {
+  type             = "ingress"
+  from_port        = 80
+  to_port          = 80
+  protocol         = "tcp"
+  cidr_blocks      = ["0.0.0.0/0"]
+  ipv6_cidr_blocks = ["::/0"]
+
+  security_group_id = aws_security_group.alb.id
+}
+
+resource "aws_security_group_rule" "alb_https_ingress" {
+  type             = "ingress"
+  from_port        = 443
+  to_port          = 443
+  protocol         = "tcp"
+  cidr_blocks      = ["0.0.0.0/0"]
```

Comment:
Can we restrict this to Azavea's IP for now until we figure out our authentication story?

Reply:
Agreed. I think that we should make this a variable that can be set per environment.